[2021-07-15 08:39:50,943] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-07-14T00:00:00+00:00 [queued]>
[2021-07-15 08:39:50,949] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-07-14T00:00:00+00:00 [queued]>
[2021-07-15 08:39:50,949] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-15 08:39:50,949] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-15 08:39:50,949] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-15 08:39:50,960] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): run_after_loop> on 2021-07-14T00:00:00+00:00
[2021-07-15 08:39:50,962] {standard_task_runner.py:52} INFO - Started process 1648 to run task
[2021-07-15 08:39:50,968] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', '2021-07-14T00:00:00+00:00', '--job-id', '21', '--pool', 'default_pool', '--raw', '--subdir', '/home/karuturihemanth/airflow_env/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmphy55x8i0', '--error-file', '/tmp/tmpdv0zz9av']
[2021-07-15 08:39:50,968] {standard_task_runner.py:77} INFO - Job 21: Subtask run_after_loop
[2021-07-15 08:39:51,003] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.run_after_loop 2021-07-14T00:00:00+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-15 08:39:51,040] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2021-07-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-14T00:00:00+00:00
[2021-07-15 08:39:51,041] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-15 08:39:51,041] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo 1']
[2021-07-15 08:39:51,047] {subprocess.py:75} INFO - Output:
[2021-07-15 08:39:51,048] {subprocess.py:79} INFO - 1
[2021-07-15 08:39:51,049] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-15 08:39:51,066] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20210714T000000, start_date=20210715T083950, end_date=20210715T083951
[2021-07-15 08:39:51,083] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-15 08:39:51,099] {local_task_job.py:151} INFO - Task exited with return code 0
